frontend/src/components/audit-log/AuditEntryIcon.vue

```diff
@@ -37,7 +37,9 @@ const iconMap = {
     nodered: [
         'crashed',
         'stopped',
-        'settings.update'
+        'settings.update',
+        'flows.set',
+        'library.set'
```

Comment:
Spotted a couple of new Node-RED events I hadn't seen before, so have added their verbose definition and icon accordingly.